docker/entrypoint.sh

```diff
@@ -1,17 +1,41 @@
 #!/bin/bash
 set -x
 
+runRspec(){
```

Comment:
Delegate the how to the function then junit reporting are generated with a different name